neevaco/neeva-android
# Queue Neeva User Info Fetch after Billing updates

Queues the fetch for neeva user info after a successful acknowledgement of a purchase. 

## Checklists

### Pointers to help with review
* https://www.figma.com/file/dylvDPB69JG9U1rzZ3HQ52/Android-Premium?node-id=0%3A1&t=oGyE4dBu3wuZBgaH-0

### How was this tested?

### Manual test cases
Got a subscription acknowledged -> should see that Settings is immediately updated after a purchase. 

Cancelled a subscription. After the subscription period ends -> should see that the user is now on the Basic plan

### Screenshots and screen recordings

Got a subscription acknowledged -> should see that Settings is immediately updated after a purchase. 

Cancelled a subscription. After the subscription period ends -> should see that the user is now on the Basic plan
